cmd/translatesfx/translatesfx/otel.go

```diff
@@ -86,6 +108,49 @@ func saInfoToOtelConfig(cfg saCfgInfo) otelCfg {
 	return out
 }
 
+func appendExtensions(m map[string]interface{}, v string) {
```

Comment:
This function (and others) is deleted in the refactoring PR: https://github.com/signalfx/splunk-otel-collector/pull/527/files